src/main/java/frc/robot/subsystems/Intake.java

```diff
@@ -4,32 +4,54 @@
 
 package frc.robot.subsystems;
 
-import com.ctre.phoenix.motorcontrol.can.WPI_TalonSRX;
+import com.ctre.phoenix.motorcontrol.VictorSPXControlMode;
+import com.ctre.phoenix.motorcontrol.VictorSPXSimCollection;
+import com.ctre.phoenix.motorcontrol.can.VictorSPX;
 
-import edu.wpi.first.wpilibj.Talon;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
 import frc.robot.HardwareMap;
 
 public class Intake extends SubsystemBase {
 
-  //private WPI_TalonSRX m_armController;  
-  //private WPI_TalonSRX m_intakeController;  
-  HardwareMap map = new HardwareMap();
+  // private WPI_TalonSRX m_armController;
+  // private WPI_TalonSRX m_intakeController;
+  private VictorSPX intakeController;
+  private VictorSPX armController;
+  private VictorSPXSimCollection intakeSim;
+  private double desiredSpeed;
+
+  // HardwareMap map = new HardwareMap();
 
   /** Creates a new Intake. */
-  public Intake() {
+  public Intake(HardwareMap hardwareMap) {
+    intakeController = hardwareMap.intakeController;
+    armController = hardwareMap.armController;
+    intakeSim = intakeController.getSimCollection();
+    desiredSpeed = 0;
   }
 
   public void intake() {
-    map.intakeController.set(-0.5);
+    desiredSpeed = 0.5;
+
   }
 
   public void outtake() {
-    map.intakeController.set(0.5);
+    desiredSpeed = -0.5;
```

Comment:
The speeds for intake and outtake were reversed - have we tested to make sure that the signs are correct now?

Reply:
That can be done during the meeting today
that shouldn't be big enough to prevent merging